scripts/asm-installer/install_asm

```diff
@@ -261,6 +263,13 @@ OPTIONS:
                                       omitted if not installing optional
                                       features. To add multiple files, specify
                                       them with multiple options one at a time.
+  -r|--remote_overlay    <FILE PATH>  The location of a YAML file in
```

Comment:
Based on what you are doing below, it is technically the file name of the overlay.

Reply:
True, good point.